firefox-ios/Client/Frontend/Browser/SearchEngines/Redux/SearchEngineSelectionState.swift

```diff
@@ -0,0 +1,74 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/
+
+import Common
+import Shared
+import Redux
+
+struct SearchEngineSelectionState: ScreenState, Equatable {
+    var windowUUID: WindowUUID
+    var shouldDismiss: Bool
+    var searchEngines: [OpenSearchEngine] // Default search engine should appear in position 0
```

Comment:
```suggestion
    // Default search engine should appear in position 0
    var searchEngines: [OpenSearchEngine]
```
nit, commenst to be above, and not inline.